inc/namespace.php

```diff
@@ -130,15 +132,39 @@ function get_client_side_data() : array {
 	return $data;
 }
 
+/**
+ * Adds an async attribute to the script tag output.
+ *
+ * @param string $tag The enqueued HTML script tag.
+ * @param string $handle The script handle.
+ * @return string The script tag markup.
+ */
+function async_scripts( string $tag, string $handle ) : string {
+	global $wp_scripts;
+
+	if ( ! $wp_scripts->get_data( $handle, 'async' ) || strpos( $tag, 'async' ) !== false ) {
+		return $tag;
+	}
+
+	$tag = str_replace( '></script>', ' async></script>', $tag );
```

Comment:
Sheesh, does wp_enqueue_script not support this?

Reply:
Sure doesn't!